debugger/debugrt/host/host.c

```diff
@@ -30,6 +31,83 @@ static void spin_unlock()
     InterlockedExchange(&_lock, 0);
 }
 
+static bool raise_debugger_events()
+{
+    static bool initialized = false;
+
+    if (IsDebuggerPresent())
+    {
+        if (!initialized)
+        {
+            // If specified, override oe_debugger_contract_version from the
+            // environment.
+            char* version = getenv("OE_DEBUGGER_CONTRACT_VERSION");
+            if (version != NULL)
+            {
+                int v = 0;
+                sscanf(version, "%d", &v);
```

Comment:
Check the return value of sscanf?

Reply:
Fixed. Added check to ensure that it is 1 before using the value.